client/db/src/upgrade.rs

```diff
@@ -62,6 +68,34 @@ fn migrate_1_to_2<Block: BlockT>(db_path: &Path, _db_type: DatabaseType) -> sp_b
 	db.add_column().map_err(db_err)
 }
 
+/// Migration from version2 to version3:
+/// - The format of the stored Justification changed to support multiple Justifications.
+fn migrate_2_to_3<Block: BlockT>(db_path: &Path, _db_type: DatabaseType) -> sp_blockchain::Result<()> {
+	let db_path = db_path.to_str()
+		.ok_or_else(|| sp_blockchain::Error::Backend("Invalid database path".into()))?;
+	let db_cfg = DatabaseConfig::with_columns(V2_NUM_COLUMNS);
+	let db = Database::open(&db_cfg, db_path).map_err(db_err)?;
+
+	// Get all the keys we need to update
+	let keys: Vec<_> = db.iter(columns::JUSTIFICATION).map(|entry| entry.0).collect();
+
+	// Read and update each entry
+	let mut transaction = db.transaction();
+	for key in keys {
+		if let Some(justification) = db.get(columns::JUSTIFICATION, &key).map_err(db_err)? {
+			let justifications = sp_runtime::Justifications::from((
+				sp_finality_grandpa::GRANDPA_ENGINE_ID,
```

Comment:
I guess we could do the same thing we did for networking and just hardcode the value here. It's a migration anyway so only the value that's currently set matters.